boards/arm/colibri_imx7d_m4/board.h

```diff
@@ -9,4 +9,10 @@
 
 #include <soc.h>
 
+#define LED0_GPIO_PORT  CONFIG_GPIO_IMX_PORT_1_NAME
+#define LED0_GPIO_PIN   2
+
+#define SW0_GPIO_NAME   CONFIG_GPIO_IMX_PORT_2_NAME
+#define SW0_GPIO_PIN    26
```

Comment:
We can now get this information from dts. Can you remove these macros and add `leds` and `gpio_keys` nodes to the board dts? See frdm_k64f for an example.